src/components/modal/ModalDialog.js

```diff
@@ -110,13 +146,20 @@ class ModalDialog extends React.Component<Props> {
             divProps['aria-describedby'] = `${this.modalID}-desc`;
         }
 
+        let modalHeaderClasses = 'modal-header';
+        if (showBackButton) {
+            modalHeaderClasses += ' modal-header--with-back';
+        }
+
         return (
             <div ref={modalRef} className={classNames('modal-dialog', className)} {...divProps}>
-                <div className="modal-header">
+                <div className={modalHeaderClasses}>
                     <h2 className="modal-title" id={`${this.modalID}-label`}>
                         {title}
                     </h2>
+                    {subtitle && <p className="modal-subtitle">{subtitle}</p>}
```

Comment:
Modal is used all over the place, so we have to be pretty deliberate about how we change its API. Can we pass the subtitle in as part of a component to the `title` prop for now?